docs/custom_loading.rst

```diff
@@ -1,15 +1,51 @@
-**************
-Custom Loading
-**************
+***************
+Loading Spectra
+***************
 
-Loading From a File
+Loading from a File
 -------------------
 
 Specutils leverages the astropy io registry to provide an interface for conveniently
 loading data from files. To create a custom loader, the user must define it in
 a separate python file and place the file in their `~/.specutils` directory.
 
 
+Loading from a FITS File
+------------------------
+The :class:`specutils.Spectrum1D` object has a read method that allows you to
+load a FITS file that contains a 1D wavelength solution.
+
+The process is actually very simple. For instance, to read a
```

Comment:
Can this be shorted to: 

    A spectra with a *Linear Wavelength Solution* can be read using the `Spectrum1D.read` 
    function to parse the file name and format